project.godot

```diff
@@ -16,7 +16,7 @@ _global_script_class_icons={
 [application]
 
 config/name="PodEscape"
-run/main_scene="res://src/Levels/World.tscn"
+run/main_scene="res://src/Scenes/Main.tscn"
```

Comment:
@rluders 👀

Reply:
After I reopen the project it got it right. Crazy yah?